.eslintrc.js

```diff
@@ -1,9 +1,13 @@
 module.exports = {
   root: true,
   env: {
-    node: true
+    es2021: true
   },
-  extends: ['plugin:vue/essential', '@vue/standard', '@vue/typescript'],
+  extends: [
```

Comment:
Missing quite a few rules here from auth-web / business-create-ui etc

Reply:
Can we create another ticket for the lint upgrade? when I copy over rules from auth-web, It has generated thousands of errors/warnings, with approximately 519 needing manual fixes.

Reply:
Sounds about right